new_webapp/src/index.css

```diff
@@ -1,17 +1,40 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
 
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
-    monospace;
+  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New', monospace;
 }
 
 .badge {
   float: left
+}
+
+.hplus_font {
+    text-align: left;
+    margin-top: 40px;
+}
+
+.title {
+    text-align: left;
+    margin-top: -10px;
+}
+
+.description {
+    text-align: left;
+    margin-top: 15px;
+}
+
+.col-10 {
+    text-align: center;
+}
+
+input {
```

Comment:
Can this be done with a class instead of a change to the whole `input` tag?

Reply:
Fixed